libcloud/compute/drivers/ec2.py

```diff
@@ -2407,8 +2407,15 @@ def create_volume_snapshot(self, volume, name=None):
 
         return snapshot
 
-    def list_volume_snapshots(self, snapshot):
-        return self.list_snapshots(snapshot)
+    def list_volume_snapshots(self, volume):
+        """List all VolumeSnapshot objects for the given StorageVolume
+
+        :param volume: the StorageVolume to filter on
+        :return: a list of all VolumeSnapshots for the given StorageVolume
+        :rtype: [VolumeSnapshot]
```

Comment:
Actually, you can remove this docstring all together since the method in the base class already has a doctring and this method signature matches the base method's signature.